### PR TITLE
Js tasks/arrays

### PR DESCRIPTION
**Pull request by BeataKwiecinska**

### CODE REVIEW

File: codewars/8kyu/arrayFilter.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+let numbers = [1, 3, 5, 1, 8, 9, 11];
+let usersAge = [12, 23, 18, 13, 45];
+let names = ['Matt', 'Beata', 'Ben', 'Tom', 'Zuza'];
+
+function checkNameLength(name){
+    return name.length > 3;
+}
```
Comment (DabTheMatt) [inline codewars/8kyu/arrayFilter.js:7]: try to leave a space line above and below the function for better code readability

File: codewars/8kyu/arrayMap.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+let numbers = [1, 3, 5, 7, 8, 9, 11];
+let numbersMaped = [];
+numbersMaped = numbers.map(function(el) {
+    return el * 2;
+})
+
```
Comment (DabTheMatt) [inline codewars/8kyu/arrayMap.js:6]: there should be no more than one line of space between code blocks

File: codewars/8kyu/convertToString.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+function booleanToString2(b){
+  }
```
Comment (DabTheMatt) [inline codewars/8kyu/convertToString.js]: unnecessary space at the beginning of the line

File: codewars/8kyu/convertToString.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+
+  booleanToString = be => `${be}`;
```
Comment (DabTheMatt) [inline codewars/8kyu/convertToString.js]: unnecessary space at the beginning of the line

**Review by DabTheMatt — CHANGES_REQUESTED** [review]
@BeataKwiecinska Well done, a few small things to improve

File: codewars/8kyu/arrayFilter.js (hunk 1 above)
Comment (BeataKwiecinska) [reply codewars/8kyu/arrayFilter.js:7]: Done

File: codewars/8kyu/arrayMap.js (hunk 2 above)
Comment (BeataKwiecinska) [reply codewars/8kyu/arrayMap.js:6]: Done

File: codewars/8kyu/convertToString.js (hunk 3 above)
Comment (BeataKwiecinska) [reply codewars/8kyu/convertToString.js]: Done

File: codewars/8kyu/convertToString.js (hunk 4 above)
Comment (BeataKwiecinska) [reply codewars/8kyu/convertToString.js]: Done

**Review by DabTheMatt — APPROVED** [review]

**Pull request merged**